spec/weaviate/schema_spec.rb

```diff
@@ -76,6 +76,30 @@
       )
       expect(response.dig("class")).to eq("Question")
     end
+
+    context "when auto_tenant_creation passed" do
+      before do
+        @captured_request = nil
+        allow_any_instance_of(Faraday::Connection).to receive(:post) do |_, path, &block|
+          expect(path).to eq("schema")
+          req = OpenStruct.new(body: {})
+          block.call(req)
+          @captured_request = req
+          response
+        end
+      end
```

Comment:
VCR or WebMock would simplify this a ton, but this is essentially grabbing the `req` object yielded to Faraday so we can check that `multiTenancyConfig` is set up correctly.